src/lib.js

```diff
@@ -1923,7 +1923,7 @@ export const getStats = async (
     output.slayer_xp = 0;
 
     for (const slayer in slayers) {
-      if (slayers[slayer]?.level?.currentLevel == undefined) {
+      if (!slayers[slayer]?.level?.currentLevel) {
```

Comment:
don't do this. the reason it is `== undefined` is because in this case you are really checking for nullishness not truthiness.

although it might no matter in this case you should avoid checking the truthiness of numbers because `0` is falsie

other things that sometimes work are `x > 0`, `typeof x == "number"`, and `x in obj`